cmd/runner/cmd.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ollama/ollama/llama"
+	"github.com/ollama/ollama/llama/runner"
+)
+
+func main() {
+	rootCmd := &cobra.Command{}
```

Comment:
RE removing `cobra`, we can probably create a `.Execute` function or similar in the `runner` package and call that here.